docs/development/testguide.rst

```diff
@@ -25,6 +25,108 @@ accessed through the ``python setup.py test`` command.
     The ``pytest`` project was formerly called ``py.test``, and you may
     see the two spellings used interchangeably in the documentation.
 
+Testing Dependencies
+********************
+
+As of Astropy 3.0, the dependencies used by the Astropy test runner are
```

Comment:
the version number here can probably be dropped, as it may be a bit confusing to see version 3.0 in the 2.0.x docs.